docs/requirements.txt

```
@@ -1,3 +1,4 @@
+livereload ~=2.6.3  # Needed by mkdocstrings
```

Not sure whether this is the correct approach - I would have thought mkdocstrings should bring this in automatically if it needs it.

---

This is tracked in https://github.com/mkdocstrings/mkdocstrings/issues/295, let's keep an eye on it. Please change the version pin to

```suggestion
livereload ~=2.6
```